includes/movienight

```diff
@@ -10,21 +10,22 @@ name="movienight"
 rcvar="${name}_enable"
 command="/usr/local/${name}/MovieNight -f /usr/local/${name}/settings.json"
 daemon_user="movien"
-managerPid="/var/run/${name}.manager.pid
+managerPid="/var/run/${name}.manager.pid"
 mnPid="/var/run/${name}.pid"
 
 start_cmd="daemon_start"
 stop_cmd="daemon_stop"
 status_cmd="daemon_status"
 
 daemon_start() {
-	/usr/sbin/daemon -P ${managerPid} -p ${mnPid} -r -f -u $daemon_user $command
+	cd /usr/local/${name}
+	/usr/sbin/daemon -P ${managerPid} -p ${mnPid} -r -f -S -o /var/log/${name}.log -u ${daemon_user} ${command}
 }
 
 daemon_stop() {
-	if [ -e "${managerPid}" ]; 
+	if [ -e "${mnPid}" ]; 
```

Comment:
You should stop the daemon by stopping `daemon`, which is the daemon manager, and not by stopping `MovieNight` witch is the managed daemon. To get the daemon manger PID: `cat ${managerPid}`.

Reply:
Switched.

Reply:
;-)